Presentation.Web/Controllers/External/V2/ItContracts/ItContractV2Controller.cs

```diff
@@ -0,0 +1,111 @@
+        /// <param name="nameContent">Name content filter</param>
+        /// <returns></returns>
+        [HttpGet]
+        [Route("it-contracts")]
+        [SwaggerResponse(HttpStatusCode.OK, Type = typeof(IEnumerable<ItContractResponseDTO>))]
+        [SwaggerResponse(HttpStatusCode.BadRequest)]
+        [SwaggerResponse(HttpStatusCode.Unauthorized)]
+        [SwaggerResponse(HttpStatusCode.Forbidden)]
+        public IHttpActionResult GetItContracts(
+            [NonEmptyGuid] Guid organizationUuid,
+            [NonEmptyGuid] Guid? systemUuid = null,
+            string nameContent = null,
+            [FromUri] StandardPaginationQuery paginationQuery = null)
+        {
+            if (!ModelState.IsValid)
+                return BadRequest(ModelState);
+
+            var refinements = new List<IDomainQuery<ItContract>>();
+
+            if (systemUuid.HasValue)
+                refinements.Add(new QueryBySystemUuid(systemUuid.Value));
+
+            if (!string.IsNullOrWhiteSpace(nameContent))
+                refinements.Add(new QueryByPartOfName<ItContract>(nameContent));
+
+            return _itContractService
+                .GetContractsInOrganization(organizationUuid, refinements.ToArray())
+                .Select(x => x.OrderBy(contract => contract.Id))
+                .Select(x => x.Page(paginationQuery))
+                .Select(x => x.ToList().Select(ToItContractResponseDto))
+                .Match(Ok, FromOperationError);
+        }
+
+        /// <summary>
+        /// Returns requested IT-Contract
+        /// </summary>
+        /// <param name="uuid">Specific IT-Contract UUID</param>
+        /// <returns>Specific data related to the IT-Contract</returns>
+        [HttpGet]
+        [Route("it-contracts/{uuid}")]
+        [SwaggerResponse(HttpStatusCode.OK, Type = typeof(ItContractResponseDTO))]
+        [SwaggerResponse(HttpStatusCode.BadRequest)]
+        [SwaggerResponse(HttpStatusCode.Unauthorized)]
+        [SwaggerResponse(HttpStatusCode.Forbidden)]
+        [SwaggerResponse(HttpStatusCode.NotFound)]
+        public IHttpActionResult GetItProject([NonEmptyGuid] Guid uuid)
+        {
+            if (!ModelState.IsValid)
+                return BadRequest(ModelState);
+
+            return _itContractService
+                .GetContract(uuid)
+                .Select(ToItContractResponseDto)
+                .Match(Ok, FromOperationError);
+        }
+
+        private static ItContractResponseDTO ToItContractResponseDto(ItContract contract)
+        {
+            return new()
+            {
+                Uuid = contract.Uuid,
+                Name = contract.Name,
+                ContractType = contract.ContractTypeId.HasValue ? contract.ContractType.MapIdentityNamePairDTO() : null,
+                Supplier = contract.SupplierId.HasValue ? contract.Supplier.MapIdentityNamePairDTO() : null,
+                InOperation = contract.IsInOperation(),
```

Comment:
her skal vi så have kontraktelementerne udstillet som nameIdPairDO'er men vi behøves ikke at lave et API til dem endnu for man skal ikke kunne registrere kontrakter endnu, så dét udfaldsrum laver vi API til på det tidspunkt, men for at genskabe overblikket i IT-System så skal vi aflevere dem så de selv kan svare på om det er i drift (det er godt nok kluntet at det er et udfaldsrum)